src/styles/_catalog.scss

```diff
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
+
+    &:hover {
+    transform: scale(1.2);
+    color: #34568b;
```

Comment:
you seem missing the card color and gaps between cards
not neat visually)
![Image 2022-03-31 22 45 17](https://user-images.githubusercontent.com/39308717/161137006-15b42eef-d76e-4261-8239-961f26f7c655.png)